landing/landing.css

```diff
@@ -303,14 +303,24 @@ main {
   width: 95%;
   margin: 0 auto;
   padding: 15px;
+  padding-bottom: 30px;
 }
 
 .code-block code {
   font-family: "Source Code Pro", monospace;
   white-space: pre-wrap;
 }
 
-
+#copy-button {
+  background-color: #ffffff00;
+  color: #000000aa;
+  float: right;
```

Comment:
Maybe using `position: absolute` with `position: relative` on the parent ? So you could place it on the top-right.

Reply:
If u want it on top-right that can be done super easy. I just choose to put it on the bottom. I will fix this.

Reply:
https://github.com/locness3/scratch-profile-embed/commit/a43598606713da0993b0c5d403648d540d785ff7

Reply:
I don't see how that would be any easier/better.

Reply:
it'd be better because it can be after the code in the DOM, while being in the top right visually.

Reply:
@locness3 Fair enough, here you go: https://github.com/locness3/scratch-profile-embed/pull/48/commits/d117a1e92dd787fd5f81c2a672fb456455a90590